.all-contributorsrc

```diff
@@ -842,6 +842,15 @@
       "contributions": [
         "code"
       ]
+    },
```

Comment:
@olegkorol `.all-contributorsrc` is broken in previous PRs, which is not a valid JSON. That's why all-contributors cli is failed. Please add a left brace to match this right brace. Thanks!

Reply:
Done ;)

Reply:
@chinesedfan